packages/tabs/src/spectrum-config.v2.js

```diff
@@ -248,6 +248,29 @@ const config = {
                 builder.class('spectrum-Tabs-selectionIndicator'),
             ],
         },
+        {
+            inPackage: '@spectrum-css/tabs',
+            outPackage: 'tabs',
+            fileName: 'tabs-overflow',
+            components: [
+                ...converter.enumerateAttributes(
+                    [
+                        ['spectrum-Tabs--sizeS', 's'],
+                        ['spectrum-Tabs--sizeM', 'm'],
+                        ['spectrum-Tabs--sizeL', 'l'],
+                        ['spectrum-Tabs--sizeXL', 'xl'],
+                    ],
+                    'size'
+                ),
+            ],
+            requireComponentPresence: [
+                {
+                    type: 'class',
+                    name: 'regex',
+                    regex: /spectrum-Tabs--size/,
+                },
+            ],
```

Comment:
Is it possible to pull just the few custom properties we are interested in?

Reply:
Not at current, as the tooling here is meant to have a one for one conversion between Spectrum CSS and SWC.

If the goal is to reduce duplication, you could do similar by making a `tabs-sizes` CSS file that is shared between the `overflow` and `tabs` elements: essentially, what you've done already, but take these values out of the conversions for Tabs.

Reply:
Good idea I think I'll go with that